### PR TITLE
Add nengo-loihi

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please restart ci¶

**Comment by Tobias-Fischer** [comment]
@conda-forge/help-python this is ready to be merged

File: recipes/nengo-loihi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - nengo_loihi.builder
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://www.nengo.ai/nengo-loihi
+  summary: Run Nengo models on the Loihi chip
+  license: MIT
```
Comment (dopplershift) [inline recipes/nengo-loihi/meta.yaml:41]: So the license is actually some custom one, not MIT:

> NengoLoihi is made available under a proprietary license that permits using, copying, sharing, and making derivative works from NengoLoihi and its source code for any non-commercial purpose, as long as the above copyright notice and this permission notice are included in all copies or substantial portions of the software.

> If you would like to use NengoLoihi commercially, licenses can be purchased from Applied Brain Research. Please contact info@appliedbrainresearch.com for more information.

Not sure yet what the right identifier is.

Comment (dopplershift) [reply recipes/nengo-loihi/meta.yaml:41]:
```suggestion
  license: LicenseRef-NENGO
```

Comment (Tobias-Fischer) [reply recipes/nengo-loihi/meta.yaml]: Thanks for catching that @dopplershift!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**